tests/scripts/tgg/run_tgg_unit_tests.sh

```diff
@@ -3,8 +3,11 @@
 set -eo pipefail
 
 run_tgg_tests() {
-  # Write tests here
-  echo "LOG_METAL: Fill me!"
+
+  echo "LOG_METAL: running run_tgg_unit_tests"
+
+  TT_METAL_SLOW_DISPATCH_MODE=1 ./build/test/tt_metal/unit_tests_galaxy --gtest_filter="GalaxyFixture.*:TGGFixture.*"
+  ./build/test/tt_metal/unit_tests_galaxy --gtest_filter="GalaxyFixture.*:TGGFixture.*"
```

Comment:
this looks to be failing on the TGG unit test pipeline?

Reply:
This seems to be failing on the TGG unit test pipeline because the TGG machine that the workflow is running on hasn't been properly instantiated. When I run the test suite locally on a TGG machine, all the tests pass.

Reply:
What do you see that leads to believe such? Can you point out some log lines?

Reply:
in the future, please raise an issue for runner issues before submitting PRs.
In principal it is difficult to validate functioning code without CI runs.
You can open draft PRs if you want to get ahead for review submissions etc but please make sure to check that you run tests in CI before submitting